### PR TITLE
Add CKS compatibility matrix details

### PR DESCRIPTION
**Pull request by Pearl1594**
Related to https://github.com/apache/cloudstack/pull/5863

### CODE REVIEW

**Comment by Pearl1594** [comment]
@blueorangutan docbuild

**Comment by Pearl1594** [comment]
@blueorangutan docbuild

**Review by rohityadavcloud — APPROVED** [review]

File: source/plugins/cloudstack-kubernetes-service.rst [hunk 1]
```diff
@@ -329,6 +329,22 @@ Token for dashboard login can be retrieved using the following command:
    # kubectl --kubeconfig /custom/path/kube.config describe secret $(kubectl --kubeconfig /custom/path/kube.config get secrets -n kubernetes-dashboard | grep kubernetes-dashboard-token | awk '{print $1}') -n kubernetes-dashboard
 
 
+Kubernetes compatibility Matrix
+#################################
+
++--------------+---------------------------------+-----------------------------+-------------+
+|ACS Version   |  Supported Kubernetes Versions  |  K8s Node Template          |  User       |
++==============+=================================+=============================+=============+
+| 4.14.x       | v1.11 onward (< 1.18)           | CoreOS                      | core        |
```
Comment (rohityadavcloud) [inline source/plugins/cloudstack-kubernetes-service.rst:338]: Instead of user, should we say "SSH User" or "Login User"? and simply say "CKS Template"

**Comment by Pearl1594** [comment]
@blueorangutan docbuild

**Review by shwstppr — APPROVED** [review]

**Comment by Pearl1594** [comment]
@blueorangutan docbuild

**Comment by Pearl1594** [comment]
@blueorangutan docbuild

**Comment by sureshanaparti** [comment]
Doc LGTM.

Doc-KubernetesCompatibilityMatrix

**Review by sureshanaparti — APPROVED** [review]

**Pull request merged**